docker-compose-dev.yml

```diff
@@ -1,6 +1,21 @@
 version: '3.7'
 
 services:
+  localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
+    image: localstack/localstack
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559"  # external services port range
+    environment:
+      - DEBUG=${DEBUG-}
+      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR-}
+      - DOCKER_HOST=unix:///var/run/docker.sock
+      - SERVICES=s3
+      - DATA_DIR=/tmp/localstack/data
+    volumes:
```

Comment:
Shouldn't we add this to the volume declaration at the bottom of the file?

Reply:
volumes were updated